imod/tests/test_mf6/test_mf6_rch.py

```diff
@@ -324,3 +326,118 @@ def test_clip_box(rch_dict):
     selection = rch.clip_box(x_min=10.0, x_max=20.0, y_min=10.0, y_max=20.0)
     assert selection["rate"].dims == ("y", "x")
     assert selection["rate"].shape == (1, 1)
+
+
+@pytest.mark.usefixtures("imod5_dataset")
+def test_planar_rch_from_imod5_constant(imod5_dataset, tmp_path):
+    data = deepcopy(imod5_dataset)
+    target_grid = data["khv"]["kh"]
```

Comment:
We do not test if the regridding works now... I don't think we need to that for each package, but having one where it is tested helps I think. Could you add a test where you regrid this case to a very coarse grid?

Reply:
in the latest version of this changeset we use the discretization package as the target grid, and it is much finer than the  target_grid = data["khv"]["kh"] that was used earlier. So now it is regridding.
That aside, xugrid does as far as I know the same operation, whether the target grid matches the input grid exactly or not. So it goes through the same code path no matter what target grid we provide.